site/src/components/popover.md

```diff
@@ -75,3 +75,12 @@ Popover is left aligned by default, but also can be centered or right aligned re
 	<div class="popover-content">Popover content.</div>
 </details>
 ```
+
```

Comment:
Can you add a heading here to separate horizontal and vertical alignment please?

```
### Vertical alignment
```

Reply:
And you can also add a section mentioning horizontal and vertical can be mixed up

Reply:
added